src/apprt/gtk/App.zig

```diff
@@ -82,7 +82,7 @@ transient_cgroup_base: ?[]const u8 = null,
 css_provider: *c.GtkCssProvider,
 
 /// Providers for loading custom stylesheets defined by user
-custom_css_providers: std.ArrayList(*c.GtkCssProvider),
+custom_css_providers: std.ArrayListUnmanaged(*c.GtkCssProvider),
 
```

Comment:
You can give this a default value as well if you want.

Reply:
Sure, initializing it here with a default value is fine.